### ISSUE TITLE
Build fails with VS Code Platform.io

### ISSUE THREAD

**Issue by dwherring** [Techstyleuk/SensESP-blackwater-tank-sensor-array#1]
When I attempt to build using Visual Studio Code Version: 1.85.0, Platform.io Core 6.1.11·Home 3.4.4 I see the following build failure:

```bash
Scanning dependencies...
Dependency Graph
|-- SensESP @ 2.7.0
Error: Nothing to build. Please put your source code files to the '/Users/dherring/work/sense/SensESP-blackwater-tank-sensor-array/src' folder
```

It appears the toolchain I'm using requires `main.cpp` to be in the `src` directory.  I'll submit a pull request for consideration in hopes that is helpful here.

@Techstyleuk : I should mention that having made the changes I'll submit for your consideration I was able to fully replicate the results you demonstrate in the very [helpful youtube video.](https://youtu.be/_G8A6kWUJCs?feature=shared)

**Comment by dwherring** [comment]
@Techstyleuk as expected I do not have access rights to push a branch to generate a PR. I don't know if you want to consider granting that or if you'd want my proposed changes in some other way. Basically I added the src directory, moved main.cpp there and added more content to the `.gitignore` file, nothing very exotic. Cheers.

**Comment by Techstyleuk** [comment]
@dwherring yes, I didn't include the full project but just the Main.ccp and platformio.ini and after you have downloaded the #https://github.com/SensESP/SensESP-project-template you simply swap out the two files, but mian.ccp does go in the src directory.  I will add that to the description - thanks
